Tests/SwiftFrameTests/Utility/ScreenshotDataMock.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
```

Comment:
Femto-nit: I think these and others aren't really "mocks". They don't mock behavior. Instead it's just static data. Maybe "...Fixtures" would be a more correct file name though this is really nitpicky.